app/views/content_items/_document_collection_body.html.erb

```diff
@@ -9,35 +9,28 @@
         content: group["body"],
         direction: page_text_direction %>
   <% end %>
-  <ol class="group-document-list" data-module="track-click">
-    <% @content_item.group_document_links(group).each_with_index do |link, link_index| %>
-      <li class="group-document-list-item">
-        <h3 class="group-document-list-item-title">
-          <%=
-            link_to(
-              link[:title],
-              link[:base_path],
-              data: {
-                track_category: 'navDocumentCollectionLinkClicked',
-                track_action: "#{group_index + 1}.#{link_index + 1}",
-                track_label: link[:base_path],
-                track_options: {
-                  dimension28: @content_item.group_document_links(group).count.to_s,
-                  dimension29: link[:title]
-                }
-              }
-            )
-          %>
-        </h3>
-        <ul class="group-document-list-item-attributes">
-          <li>
-            <time datetime="<%= link[:public_updated_at].iso8601 %>">
-              <%= l(link[:public_updated_at], format: :short_ordinal) %>
-            </time>
-          </li>
-          <li><%= t("content_item.schema_name.#{link[:document_type]}", count: 1) %></li>
-        </ul>
-      </li>
-    <% end %>
-  </ol>
+
+  <div data-module="track-click">
+    <%
+      items = Array.new
+      @content_item.group_document_links(group).each_with_index do |link, link_index|
+        items[link_index] = {
+          text: link[:title],
```

Comment:
There's already a method named group_document_links, could be confusing? How about create_document_links_list?